docs/gen2_Tutorial.md

```diff
@@ -429,6 +429,73 @@ and clean them up.
 
 If the cluster which which is being deleted is a cluster created using patch files, the controller first cleans the git repo where the respective patch files of the cluster are present and then it destroys all the related ArgoCD applications and clean them up.
 
+## Enabling Cluster Autoscaler in the workload cluster
+To create a gen2 cluster with autoscaler, we need:
+
+- bundle pointing to the bundle/capi-cluster-autoscaler in the arlon repository.
+- dynamic profile that contains the above bundle.
+- a basecluster manifest which has the CAPI annotations for min and max nodes set ( as a part of `preparegit` or manually add it ).
+- run `arlon cluster create` with the repo-path pointing to the basecluster manifest described in the step above, set the profile to  be the one created in step 2 and pass the `autoscaler` flag.
+
+### Bundle creation
+
+Register a dynamic bundle pointing to the bundles/capi-cluster-autoscaler in the Arlon repo.
+
+The capi-cluster-autoscaler bundle requires the name of the cluster, so that it knows what namespace in the management cluster to scan for CAPI resources. To enable the cluster-autoscaler bundle, add one more parameter during cluster creation: `srcType`. This is the ArgoCD-defined application source type (Helm, Kustomize, Directory).
+
+This example creates a bundle pointing to the bundles/capi-cluster-autoscaler in Arlon repo
+
+```shell
+arlon bundle create cas-bundle --tags cas,devel,test --desc "CAS Bundle" --repo-url https://github.com/arlonproj/arlon.git --repo-path bundles/capi-cluster-autoscaler --srctype helm
```

Comment:
The user also needs to specify a specific revision that points to a git hash, tag or stable Arlon release branch, so that the Arlon project doesn't unintentionally break CAS functionality through a git update. Have there been changes to the bundle since 0.9? If not, then v0.9 release branch is fine. Otherwise it has to be v0.10.